src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -239,11 +240,12 @@ const renderEmployeeVacationRequests = () => {
         onChange={ handleVacationTypeChange }
         label={ strings.editorContent.vacationType }
       >
-        <MenuItem value="Paid leave">{ strings.editorContent.paidLeave }</MenuItem>
-        <MenuItem value="Maternity leave">{ strings.editorContent.maternityLeave }</MenuItem>
-        <MenuItem value="Parental leave">{ strings.editorContent.parentalLeave }</MenuItem>
-        <MenuItem value="Unpaid leave">{ strings.editorContent.unpaidLeave }</MenuItem>
-        <MenuItem value="Surplus balance">{ strings.editorContent.surplusBalance }</MenuItem>
+        <MenuItem value={ VacationType.VACATION }>{ strings.editorContent.vacation }</MenuItem>
```

Comment:
Its an edge case but I would consider indenting the content of the menuItems (the string) on its own line having the Menu Item tags on their own lines above and below. Just to improve clarity/ indentation